src/aml_service/index.js

```diff
@@ -0,0 +1,86 @@
+'use strict'
+
+const yotiRequest = require('../yoti_request');
+const AmlResultClass = require('../yoti_request/aml.result').AmlResult;
```

Comment:
I believe '.' is used in `javascript` file name instead of snake_case or camelCase, I followed the same approach used in `src/proto-root`.
Also I am using `yoti_` for name-spacing as these are part of `Yoti` package, they might not belong to `Yoti`.